src/Generator/Generator.php

```diff
@@ -867,7 +867,38 @@ public function getSoapClient()
     public function getUrlContent($url)
     {
         if (strpos($url, '://') !== false) {
-            return Utils::getContentFromUrl($url, $this->getOptionBasicLogin(), $this->getOptionBasicPassword(), $this->getOptionProxyHost(), $this->getOptionProxyPort(), $this->getOptionProxyLogin(), $this->getOptionProxyPassword(), $this->getSoapClient()->getSoapClientStreamContextOptions());
+            $content = Utils::getContentFromUrl(
+                $url, 
+                $this->getOptionBasicLogin(), 
+                $this->getOptionBasicPassword(), 
+                $this->getOptionProxyHost(), 
+                $this->getOptionProxyPort(), 
+                $this->getOptionProxyLogin(), 
+                $this->getOptionProxyPassword(), 
+                $this->getSoapClient()->getSoapClientStreamContextOptions()
+            );
```

Comment:
I know the line is long but for the meantime, keep it on one line, in addition, it breaks the [CS](https://styleci.io/analyses/8Q4P6o)